src/cascadia/TerminalApp/Command.cpp

```diff
@@ -22,8 +22,10 @@ static constexpr std::string_view IterateOnKey{ "iterateOn" };
 static constexpr std::string_view CommandsKey{ "commands" };
 
 static constexpr std::string_view IterateOnProfilesValue{ "profiles" };
+static constexpr std::string_view IterateOnSchemesValue{ "schemes" };
 
 static constexpr std::string_view ProfileName{ "${profile.name}" };
+static constexpr std::string_view SchemeName{ "${scheme.name}" };

```

Comment:
nit: now that we've added a second i sorta want to name them `XxxNameToken`

Reply:
you've got it boss